lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -285,18 +285,11 @@ def test_contourf3d_extend(fig_test, fig_ref, extend, levels):
     # Z is in the range [0, 8]
     Z = X**2 + Y**2
 
-    # Manually set the over/under colors to be the end of the colormap
-    cmap = mpl.colormaps['viridis'].copy()
-    cmap.set_under(cmap(0))
-    cmap.set_over(cmap(255))
-    # Set vmin/max to be the min/max values plotted on the reference image
-    kwargs = {'vmin': 1, 'vmax': 7, 'cmap': cmap}
```

Comment:
This cmap acrobatics is actually unneeded. If the colormap does not have extremes, as in the case of viridis, the values used to render over/under are actually the ends of the colormap. So setting them explicitly does not have an impact on the rendered image.